py/desiutil/dust.py

```diff
@@ -17,6 +17,30 @@
 from .log import get_logger
 log = get_logger()
 
+def extinction_total_to_selective_ratio(band, photsys) :
+    """Return the linear coefficient R_B = A(B)/E(B-V) where
+    A(B) = -2.5*log10(transmission in B band), for band B in 'G','R' or 'Z',
+    the optical bands of the legacy surveys. photsys = 'N' or 'S' specifies
+    the survey (BASS+MZLS or DECALS).  E(B-V) is interpreted as SFD.
+
+    Args:
+        band : 'G', 'R' or 'Z'
+        photsys : 'N' or 'S'
+
+    Returns:
+        scalar, total extinction A(band) = -2.5*log10(transmission(band))
+    """
+
+    # fitted from the imaging data (using fibermaps of 2020/03/15)
```

Comment:
This comment isn't very clear on where these data actually come from.  Can additional details be added?

Also, are these numbers likely to change over the course of the survey?  If so, do they have to be hard-coded in this file?

Reply:
Lots of meetings today so interleaving coding, commenting, and delays.  Let's *not* move params into a separate file unless we find that is really needed.  e.g. simply moving them into yaml file within desiutil would still require a new desiutil tag to update them so is effectively the same thing as hardcoding unless they need to be used in a different function to (I don't think they do).

Reply:
OK, I get that, but can some details about where the parameters actually come from would be very useful for long-term maintenance.

Reply:
Also, it would be great if a change log entry could be added to the PR.